src/App.tsx

```diff
@@ -15,36 +16,96 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  alphabetic = 1,
+  length = 2,
+}
+
+export const sortGoods = (goods, params) => {
```

Comment:
The `sortGoods` function should have explicit type annotations for its parameters and return type. Consider defining the types for `goods` and `params`, and specifying the return type of the function.

Reply:
> The `sortGoods` function should have explicit type annotations for its parameters and return type. Consider defining the types for `goods` and `params`, and specifying the return type of the function.

Good point. You don't need to add annotation for the return, typescript will do it for you, but you have to add types for params as in your case they have `any` type and it would be the same as you will write your code in javascript